src/orion/core/__init__.py

```diff
@@ -30,7 +30,7 @@
     'tsirif': ('Christos Tsirigotis', 'tsirif@gmail.com'),
     'bouthilx': ('Xavier Bouthillier', 'xavier.bouthillier@gmail.com'),
 }
-__url__ = 'https://github.com/mila-udem/orion'
```

Comment:
(I am not sure why I cannot comment on something else apart from the changes, but I think what I am about to write is relevant here in this file and PR)

We should probably apply corresponding changes to the private-magic variables above this one, with respect to the `MILA` -> `Epistimio` change.

Reply:
Indeed. I did it in Kleio but forgot here. I'll add this and rebase.

Reply:
Done

Reply:
@tsirif Ping :)

Reply:
@tsirif Ping :)